### PR TITLE
german translation update

### PR DESCRIPTION
**Pull request by Laserlicht**

### CODE REVIEW

File: files/lang/de.po [hunk 1]
```diff
@@ -185,7 +185,7 @@ msgid "DIFFICULTY"
 msgstr "SCHWIERIG."
 
 msgid "VIEW INTRO"
-msgstr ""
+msgstr "INTRO ANZ."
```
[the remark this answers is not in the document]
Comment (Laserlicht) [reply files/lang/de.po]: Okay. It's close, but it fits. ;)

grafik

[the remark this answers is not in the document]
Comment (Laserlicht) [reply files/lang/de.po]: That would be "VIDEO ANZEIGEN" (or maybe the short form "VIDEO ZEIGEN") / "VIDEO ABSPIELEN" / "FILM ANZEIGEN". German is usually quite long. :)

File: files/lang/de.po [hunk 2]
```diff
@@ -3545,264 +3544,272 @@ msgstr "Person:"
...
+"Wird für die spezielle Bearbeitung von Monstern, Helden und Städten "
+"verwendet."
 
 msgid "Allows you to draw streams by clicking and dragging."
-msgstr ""
+msgstr "Ermöglicht das Zeichnen von Streams durch Klicken und Ziehen."
 
 msgid "Stream Mode"
-msgstr ""
+msgstr "Stream-Modus"
```
[the remark this answers is not in the document]
Comment (Laserlicht) [reply files/lang/de.po]: Yes, you're right. Thanks :)

**Comment by Laserlicht** [comment]
> Hi, @Laserlicht. Could you please solve the merge conflicts?

done :)

**Comment by Laserlicht** [comment]
closes #7496

**Comment by Laserlicht** [comment]
Hi @zenseii 

Shouldn't we close this PR here and make a separate one only for the buttons?

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Laserlicht , thank you so much for this update!

**Pull request merged**